src/Bedrock.Framework.Experimental/Protocols/Memcached/Expiration.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.Memcached
+{
+    public readonly struct Expiration
+    {
+        public readonly uint Value { get; }
```

Comment:
Should the public type really be `uint` or is it better / more compliant to use `int` here?